r/R/dataset-write.R

```diff
@@ -54,6 +54,40 @@
 #' - `null_fallback`: character to be used in place of missing values (`NA` or
 #' `NULL`) when using Hive-style partitioning. See [hive_partition()].
 #' @return The input `dataset`, invisibly
+#' @examples
+#' # We can partition by one more variables, say, cyl and gear in mtcars dataset
```

Comment:
This is a personal preference, so you don't have to do it, but I think it's nice to have the comments just above each line of code explaining exactly what that code is doing in examples. So here start with `# We can write datasets partitioned by the values in a column (here: "cyl")` then have the first `write_dataset()` line. Then have another line `# We can also partition by the values in multiple columns.` and then have the second `write_dataset()`